### DISCUSSION TITLE
Welcome to Goodtime Productivity Discussions!

### DISCUSSION THREAD

**Discussion by adrcotfas** [adrcotfas/goodtime#294] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other people interested in Goodtime Productivity and
  * ask questions you’re wondering about.
  * Share ideas about what can be improved in the app.
  * engage with other people using the app.

**Comment by HighPriest** [comment]
How is the work on v3 going? @adrcotfas   
Any details on a roadmap, completion status, maybe proposed tasks for interested contributors?

**Comment by KhaledAlabyad** [comment]
it will be great to make gestures customizable specially swipe up gesture to change the time being added instead of just 1 min

**Comment by HerrPischinger** [comment]
Is there a roadmap for new features? Getting forced to lay the phone upside down on the table like in "Focus to-do" would be really nice. Great app!

**Reply by adrcotfas** [reply]
@HerrPischinger I don't have a public roadmap here but the next priority is releasing the iOS version of this app along with some improvements for time profiles.  
I'm open for suggestions and if you open a new discussion, others can upvote.

**Reply by HerrPischinger** [reply]
As someone who uses an android phone and an iPad this sounds great! Thanks for the tip. Ill think about it :)

**Comment by HighPriest** [comment]
Just a word of appreciation @adrcotfas and thanks for great software. You are a really talented programmer.  
I've been away from programming for a while myself and I really miss your work. I hope to get back to using it again, soon.

**Comment by chrsdk** [comment]
I just discovered this app. It's so well done. A minimalistic android widget would make it complete. I would like to have it on my home screen to start it whenever I pick up the phone. Without a widget I forget to use it.

**Reply by HighPriest** [reply]
I have it fully take over my start page. I mean... I start it and every 25 minutes it is there, nagging me to re-think my life choices.  
Are you sure that a widget, of some sort, is necessary?

**Reply by chrsdk** [reply]
When I start it, and I have to think about starting it, since it's not on my homepage, it runs in the background and gives me a notification after 25m. I have to swipe up and browse through my open windows to find Goodtime. If i miss the notification, then it just accumulates "idle time". If it would sit as a widget on my home screen, then I would see it all the time and not forget that it's there.

Maybe it behaves different on different devices? I am running Android 15 on a Samsung.
